tests/Hippo.FunctionalTests/FakeBindleService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Hippo.Application.Common.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hippo.FunctionalTests
```

Comment:
Can you please change this to use the new file-scoped namespace declaration as we do elsewhere?

https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-10#file-scoped-namespace-declaration